store/sqlstore/channel_store.go

```diff
@@ -2179,8 +2179,6 @@ func (s SqlChannelStore) UpdateLastViewedAt(channelIds []string, userId string,
 func (s SqlChannelStore) CountPostsAfter(channelId string, timestamp int64, userId string) (int, int, error) {
 	joinLeavePostTypes := []string{
 		// These types correspond to the ones checked by Post.IsJoinLeaveMessage
-		model.PostTypeJoinLeave,
```

Comment:
Ahh bummer .. this is needed for the SQL query to work with old posts. So it seems like we need that in server too. @hmhealey. I'll revert the changes.